cmux_test.go

```diff
@@ -180,8 +220,7 @@ func TestClosed(t *testing.T) {
 	lis := mux.Match(Any()).(muxListener)
 	close(lis.donec)
 	mux.serve(closerConn{})
-	_, err := lis.Accept()
-	if _, ok := err.(errListenerClosed); !ok {
-		t.Errorf("expected errListenerClosed got %v", err)
+	if _, err := lis.Accept(); err != ErrListenerClosed {
```

Comment:
I know, this is the only instance; but only for now. I have plans to add more of them later on. thanks.